kork/spinnaker-dependencies/spinnaker-dependencies.gradle

```diff
@@ -46,9 +46,9 @@ dependencies {
    * For example, `junit-bom` and `jackson-bom` will win out over the versions of JUnit and Jackson
    * specified by `spring-boot-dependencies`.
    */
-   // Log4shell safeguard.  Per analysis, log4j-core is not included in dependencies, but this would prevent transitive inclusion of it by extension
-   // platforms.   Doing 2.16.0 which completely removes message lookups AND sets jndi to disabled by default
-   // 2.16.0 is subject to CVE-2021-45105.  2.17.0 is subject to CVE-2021-44832, so use >= 2.17.1.
+  // Springdoc includes a newer version of Jackson, which conflicts with the SnakeYaml pin below
+  // Jackson 2.15+ requires SnakeYaml 2.x, last Jackson version with SnakeYaml 1.31 compat is 2.14.0
```

Comment:
We're specifying snakeyaml 1.33 below, so this comment doesn't really make sense.

Plus also [spring boot 3.0.13][(https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/3.0.13/spring-boot-dependencies-3.0.13.pom) specifies jackson-bom 2.14.3, so let's specify that.

Reply:
Updated the `jackson-bom `to 2.14.3 to align with Spring Boot 3.0.13 and ensure compatibility with SnakeYAML 1.33. Here is the commit https://github.com/rahul-chekuri/spinnaker/commit/07737cfa

I tried removing the `jackson-bom`, but Gate tests started failing, so I’ve kept it in place.